pkg/openshift/certgen/templates/master/tmp/ansible/azure-ocp-deploy.yml

```diff
@@ -3,6 +3,10 @@
   gather_facts: false
```

Comment:
Shouldn't the name of this playbook be updated?

Reply:
Updated, thanks.